subsys/mgmt/fmfu/src/mgmt_fmfu.c

```diff
@@ -170,7 +170,8 @@ static int mgmt_firmware_upload(struct mgmt_ctxt *ctx)
 						 packet.data_len,
 						 packet.data);
 		if (rc != 0) {
-			LOG_ERR("Error in writing data: %d", errno);
+			LOG_ERR("Error in writing data, rc: %d, errno: %d", rc,
```

Comment:
Maybe this should be in the previous commit.

Reply:
yes